lib/maxsub.gi

```diff
@@ -542,7 +542,7 @@ local id,m,epi;
     # following is stopgap for L
     id:=DataAboutSimpleGroup(G);
     if id.idSimple.series="A" then
-      Info(InfoLattice,1,"Alternating recognition needed!");
+      Info(InfoPerformance,1,"Alternating recognition needed!");
     elif id.idSimple.series="L" then
```

Comment:
@hulpke: It actually will not work - `InfoPerformance` has level 1 when GAP starts. You can check this with `make testinstall` - the 1st of the log files with have these messages.